deployment/docker-compose/pretix/etc/pretix.cfg

```diff
@@ -0,0 +1,20 @@
+[pretix]
+instance_name=My pretix installation
+url=http://127.0.0.1
+currency=EUR
+datadir=/data
+
+[database]
+backend=mysql
+name=pretix
+user=pretix
+password=pretix
+host=db
+
+[redis]
+location=redis://redis/0
+session=true
```

Comment:
I think this needs to be sessions, according to https://github.com/pretix/pretix/blob/3c3e59e932080dffe9dca18156693a8b6573a9dc/src/pretix/settings.py#L182

Also see https://github.com/ecobytes/pretix/blob/master/deployment/docker-compose/pretix/etc/pretix.cfg#L33

Reply:
true, nice catch :+1: